frontend/package.json

```diff
@@ -14,19 +14,23 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "typecheck": "nuxt typecheck .",
-    "deploy": "npm run generate && firebase --project=default deploy",
-    "deploy:hosting": "npm run generate && firebase --project=default deploy --only hosting"
+    "deploy": "npm run generate &&  npx -y firebase-tools --project=default deploy",
+    "deploy:hosting": "npm run generate && npx -y firebase-tools --project=default deploy --only hosting"
```

Comment:
are these redundant since the makefile does the same thing?

Reply:
it seems the makefile is unused in the ci/cd pipeline, i'll clean that up when i update functions